samples/ios/GleanSamplesWorkspace.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there docs on workspaces in Xcode? I want to understand what you did here.

Reply:
The only docs I found were old but describe the idea: [Xcode Workspaces](https://developer.apple.com/library/archive/featuredarticles/XcodeConcepts/Concept-Workspace.html).  

The tl/dr is that it allows us to have both the app and the framework open in the same instance of xcode.  The projects are still treated separately but it made it easier to include the framework in the app and smooths the overall workflow where we don't have to switch between two Xcode instances.

Reply:
👍